NSTTutorials/ResponseLoggerInjections/src/test/java/com/ebay/nst/tutorials/rest/responseloggerinjections/ResponseLoggerInjectionsTest.java

```diff
@@ -0,0 +1,50 @@
+package com.ebay.nst.tutorials.rest.responseloggerinjections;
+
+import com.ebay.nst.NSTServiceTestRunner;
+import com.ebay.nst.NSTServiceWrapperProcessor;
+import com.ebay.nst.tutorials.sharedtutorialutilities.rest.CanadaHoliday;
+import com.ebay.softassert.EbaySoftAssert;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class ResponseLoggerInjectionsTest implements NSTServiceTestRunner {
+
+    private static final String RESPONSE_LOGGER_INJECTIONS_PATH = "src/test/java/responseloggerinjectionstutorial";
+    private static final String GENERATED_MOCK_PATH = RESPONSE_LOGGER_INJECTIONS_PATH + "/ResponseLoggerInjectionsTest_exampleResponseLoggerInjectionMockGenerationTest_0_ResponseLoggerInjectionsWrapper.har";
+
+    @BeforeClass
+    // Remove any prior mocks that were generated by running the test method below.
+    public void removeExistingMock() {
+        File existingMock = new File(GENERATED_MOCK_PATH);
+        if (existingMock.exists()) {
+            existingMock.delete();
+            System.out.printf("Removed existing mock at: %s%n", GENERATED_MOCK_PATH);
+        }
+    }
+
+    @Test
+    public void exampleResponseLoggerInjectionMockGenerationTest() throws Exception {
+        NSTServiceWrapperProcessor serviceProcessor = new NSTServiceWrapperProcessor();
+
+        // Send a GET /api/v1/holidays/{holidayId} request.
+        ResponseLoggerInjectionsWrapper restServiceWrapperWithResponseLoggerInjection = new ResponseLoggerInjectionsWrapper(CanadaHoliday.CANADA_DAY);
```

Comment:
Your logger could take a parameter to allow the consumer of the tutorial to turn off the injection so they can see the mock with and without the injection in play.

Reply:
Done in 3d43f0926bdcdf8d1cddbe7d5466662922cf51bb